ExtraRoles/SetInfectedPatch.cs

```diff
@@ -76,35 +84,32 @@ public static void Postfix(Il2CppReferenceArray<GameData.PlayerInfo> JPGEIBIBJPJ
             {
                 var JokerRandom = rng.Next(0, crewmates.Count);
                 crewmates[JokerRandom].getModdedControl().Role = "Joker";
-                byte JokerId = crewmates[JokerRandom].PlayerId;
+                var JokerId = crewmates[JokerRandom].PlayerId;
                 crewmates.RemoveAt(JokerRandom);
 
                 System.Console.WriteLine("Spawning Joker with PlayerID = " + JokerId);
 
-                writer = AmongUsClient.Instance.StartRpcImmediately(PlayerControl.LocalPlayer.NetId, (byte)CustomRPC.SetJoker, Hazel.SendOption.None, -1);
+                writer = AmongUsClient.Instance.StartRpcImmediately(PlayerControl.LocalPlayer.NetId,
+                    (byte) CustomRPC.SetJoker, Hazel.SendOption.None, -1);
                 writer.Write(JokerId);
                 AmongUsClient.Instance.FinishRpcImmediately(writer);
             }
 
             localPlayers.Clear();
             localPlayer = PlayerControl.LocalPlayer;
-            foreach (PlayerControl player in PlayerControl.AllPlayerControls)
+            foreach (var player in PlayerControl.AllPlayerControls)
             {
                 if (player.Data.IsImpostor)
```

Comment:
if (!(player.Data.IsImposter || player.isPlayerRole("Joker")))
    localPlayers.Add(player);

Reply:
"continue" is redundant

Reply:
Improved the loop and replaced the two distinct checks with an if.